python/pyspark/pandas/strings.py

```diff
@@ -2316,7 +2316,7 @@ def zfill(self, width: int) -> "ps.Series":
         left). 1000 remains unchanged as it is longer than width.
 
         >>> s.str.zfill(3)
-        0     0-1
+        0     -01
```

Comment:
This bug was fixed in pandas 1.5.0
https://github.com/pandas-dev/pandas/issues/20868

Reply:
+1